src/test/kotlin/CheckerTests.kt

```diff
@@ -0,0 +1,73 @@
+import org.junit.jupiter.api.Test
+import kotlin.random.Random
+
+class CheckerTests {
+
+    @Test
+    fun checkAllCorrect() {
+        for (sequenceLength in MIN_SEQ_LENGTH..MAX_SEQ_LENGTH) {
+            for (colors in MIN_COLORS..MAX_COLORS) {
+                val code = List(sequenceLength) { Random.nextInt(0, colors).toString() }
+                val checker = Checker(code)
+                checker.checkGuess(code).apply {
+                    assert(correct == sequenceLength)
```

Comment:
generalnie to raczej nie należy używać "surowego" assert, bo
1. działa tylko przy włączonym parametrze -ea (gradle akurat go ustawia dla testów, ale wystraszyłem się, że te testy tak naprawdę nic nie sprawdzają...)
2. lepiej używać odpowiednich asercji z junita, w tym przypadku assertEquals, bo wypisują jaki był actual wynik, a jaki był oczekiwany. Surowe assert wypisze po prostu `java.lang.AssertionError: Assertion failed`